utils/crypto.go

```diff
@@ -0,0 +1,107 @@
+package utils
+
+import (
+	"bytes"
+	"crypto/rand"
+	"errors"
+	"io/ioutil"
+	"math/big"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+)
+
+// decrypt uses PGP to decrypt symmetrically encrypted and armored text
+// with the provided password.
+func Decrypt(text []byte, password []byte) (decryptedText []byte, err error) {
+	decbuf := bytes.NewBuffer(text)
+
+	armorBlock, err := armor.Decode(decbuf)
+	if err != nil {
+		return
+	}
+
+	failed := false
+	prompt := func(keys []openpgp.Key, symmetric bool) ([]byte, error) {
+		// If the given passphrase isn't correct, the function will be called again, forever.
+		// This method will fail fast.
+		// Ref: https://godoc.org/golang.org/x/crypto/openpgp#PromptFunction
+		if failed {
+			return nil, errors.New("Unable to unlock safe with provided password")
+		}
+
+		failed = true
+
+		return password, nil
+	}
+
+	md, err := openpgp.ReadMessage(armorBlock.Body, nil, prompt, nil)
```

Comment:
Would you mind opening an issue for this one? If we expose control of encryption, I'd like to consider the design of flags and/or `config` files, ciphers, etc.